### ISSUE TITLE
Doesn't launch on macOS Sierra

### ISSUE THREAD

**Issue by DEGoodmanWilson** [Gnurou/tagainijisho#183]
Crashes immediately on launch, with the following in left in the Console

```
10/9/16 9:38:57.289 AM com.apple.xpc.launchd[1]: (com.apple.xpc.launchd.oneshot.0x1000003c.Tagaini Jisho[99743]) Service exited due to signal: Killed: 9
```

**Comment by Gnurou** [comment]
Mm, I guess the SDK used to build is too old for Sierra? MacOS has a terrible record at backward compatibility...

We need someeone with Sierra to attempt a build, as I can only produce binaries for Linux and Windows sadly.

**Comment by DEGoodmanWilson** [comment]
I gave it a go, but running into issues around QT and standard headers. Will post the result here later. This uses QT4, right?

**Comment by Gnurou** [comment]
At the moment, yes - although I will switch to Qt5 soonish. Appreciate your help on this!

**Comment by firatakandere** [comment]
I can confirm that it works on High Sierra though.

**Comment by Gnurou** [comment]
Probably fixed as a result of the Qt5 update and 1.2.x releases - closing.
